packages/react/src/components/PhoneNumberInput/constants/countries.ts

```diff
@@ -0,0 +1,1236 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export interface Country {
+  code: string;
+  dialCode: string;
```

Comment:
Is dialcode a common term?

Reply:
IMO, country code is the most commonly used term.

https://countrycode.org/

Reply:
Country Calling Codes or Country Dial-in Codes seems to be the standard. 
https://en.wikipedia.org/wiki/List_of_country_calling_codes